packages/fxa-auth-server/lib/senders/emails/templates/subscriptionUpgrade/index.mjml

```diff
@@ -0,0 +1,37 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+<mj-section css-class="mb-6">
+  <mj-group>
+    <mj-column>
+      <mj-image src="<%- productIconURLOld %>" alt="<%- productNameOld %>" title="<%- productNameOld %>" css-class="product-icon">
+      </mj-image>
+    </mj-column>
+    <mj-column>
+      <mj-image src="<%- productIconURLNew %>" alt="<%- productNameNew %>" title="<%- productNameNew %>" css-class="product-icon">
+      </mj-image>
+    </mj-column>
+  </mj-group>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="subscriptionUpgrade-title" data-l10n-args="<%= JSON.stringify({productNameNew}) %>">Thank you for upgrading to <%- productNameNew %>!</span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="subscriptionUpgrade-content" data-l10n-args="<%= JSON.stringify({paymentAmountNew, paymentAmountOld, paymentProrated, productNameNew, productNameOld, productPaymentCycle}) %>">
+        You have successfully upgraded from <%- productNameOld %> to <%- productNameNew %>.
+        <br><br>
```

Comment:
For some reason, I'm not seeing these `<br><br>`s being rendered properly 🤔 

<img width="704" alt="image" src="https://user-images.githubusercontent.com/13018240/144485233-f2ea5995-8683-4f78-9762-2af1ab4c6260.png">

Though... I really am not a fan of a double line break like this anyway because, on the web at least, many screen readers will read this as one large blob rather than reading them as separate elements which would better match a sighted user's experience. I think we probably want separate `mj-text` elements for each paragraph.